app/views/comments/_comment_form.html.erb

```diff
@@ -64,46 +64,39 @@
+          <dt><%= f.label "name_for_#{commentable.id}", t(".guest_name") %></dt>
           <dd>
-            <%= f.text_field :name, :id => "comment_name_for_#{commentable.id}" %>
-            <%= live_validation_for_field("comment_name_for_#{commentable.id}", :failureMessage => ts('Please enter your name.')) %>
+            <%= f.text_field :name, id: "comment_name_for_#{commentable.id}" %>
+            <%= live_validation_for_field("comment_name_for_#{commentable.id}", failureMessage: t(".guest_name_failure")) %>
           </dd>
-          <dt><%= f.label "email_for_#{commentable.id}", ts("Guest email: ") %></dt>
+          <dt><%= f.label "email_for_#{commentable.id}", t(".guest_email") %></dt>
           <dd>
-            <%= f.text_field :email, :id => "comment_email_for_#{commentable.id}" %>
-            <%= live_validation_for_field("comment_email_for_#{commentable.id}", :failureMessage => ts('Please enter your email address.')) %>
+            <%= f.text_field :email, id: "comment_email_for_#{commentable.id}" %>
+            <%= live_validation_for_field("comment_email_for_#{commentable.id}", failureMessage: t(".guest_email_failure")) %>
           </dd>
         </dl>
         <p class="footnote">(<%= allowed_html_instructions %>)</p>
       <% end %>
 
       <p>
         <% content_id = "comment_content_for_#{commentable.id}" %>
-        <label for="<%= content_id %>" class="landmark"><%= ts("Comment") %></label>
-        <%= f.text_area :comment_content, :id => content_id, :class => "comment_form observe_textlength", :title => ts("Enter Comment") %>
+        <label for="<%= content_id %>" class="landmark"><%= t(".comment_action") %></label>
+        <%= f.text_area :comment_content, id: content_id, class: "comment_form observe_textlength", title: t(".comment_field_title") %>
         <input type="hidden" id="controller_name_for_<%= commentable.id %>" name="controller_name" value="<%= @controller_name ||= controller.controller_name %>" />
       </p>
       <%= generate_countdown_html("comment_content_for_#{commentable.id}", ArchiveConfig.COMMENT_MAX) %>
-      <%= live_validation_for_field("comment_content_for_#{commentable.id}",
-              :failureMessage => ts('Brevity is the soul of wit, but we need your comment to have text in it.'),
-              :maximum_length => ArchiveConfig.COMMENT_MAX,
-              :tooLongMessage => ts("must be less than %{count} characters long.", :count => ArchiveConfig.COMMENT_MAX)) %>
-      <p class="submit actions">
-        <%= f.submit button_name, :id => "comment_submit_for_#{commentable.id}", data: {disable_with: ts("Please wait...")} %>
+      <%= live_validation_for_field "comment_content_for_#{commentable.id}",
+            failureMessage: t(".comment_too_short"),
+            maximum_length: ArchiveConfig.COMMENT_MAX,
+            tooLongMessage: t(".comment_too_long", count: ArchiveConfig.COMMENT_MAX) %>
+    <p class="submit actions">
```

Comment:
I think this needs to be indented two spaces more, assuming ERB Lint doesn't complain.